realm/realm-jni/src/io_realm_internal_table.cpp

```diff
@@ -1693,3 +1693,20 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_Table_nativeHasSameSchema
 {
     return *TBL(thisTablePtr)->get_descriptor() == *TBL(otherTablePtr)->get_descriptor();
 }
+
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_Table_nativeVersion(
+        JNIEnv* env, jobject, jlong nativeTablePtr)
+{
+    bool valid = (nativeTablePtr != 0);
+    if (valid) {
+        if (!TBL(nativeTablePtr)->is_attached()) {
+            ThrowException(env, TableInvalid, "Table is closed, and no longer valid to operate on.");
```

Comment:
same in `Java_io_realm_internal_TableView_nativeSync` &  `view_valid_and_in_sync` what do you suggest ?